core/src/main/java/org/elasticsearch/monitor/os/OsStats.java

```diff
@@ -152,8 +156,9 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
     public static class Cpu implements Streamable {
+
         short percent = -1;
-        double loadAverage = -1;
+        double[] loadAverage = new double[] { -1, -1, -1 };
```

Comment:
what do we need to initialize this to a strange value vs `null`? Is `null` not good enough? I see confusing code like this in all these `Streamable`s.... to me it seems like the whole entire Streamable API is broken. I dont understand why we have readFrom() at all, vs. say using an abstract class with a ctor that takes Streamable, and thats just how things are deserialized. Then these various classes could be immutable and there would be much less confusion?